refract-spec.md

```diff
@@ -117,29 +474,307 @@ This example will show a JSON object represented as both a normal Refract Elemen
 }
 ```
 
-##### Refract Element Representation
+In Refract, this object is expanded to a Object Element.
+
+```json
+{
+  "element": "object",
+  "content": [
+    {
+      "key": {
+        "element": "string",
+        "content": "foo"
+      },
+      "value": {
+        "element": "string",
+        "content": "bar"
+      }
+    }
+  ]
+}
+```
 
-It can be expressed in the normal Refract element structure.
+## Referencing Elements
+
+The referencing elements are provided as part of the base specification for the purpose of identifying and referencing elements.
+
+### ID (Element)
+
+The `id` element is a way to identify an element in a document. IDs MUST be unique throughout a document.
+
+#### Properties
+
+- `element` id (string, fixed)
+- `content` (string) - Unique identifier
+
+#### Example
+
+The `id` element is used here in the meta attributes to give an ID of `foo` to this specific instance of a string with the value of "bar."
 
 ```json
 {
-  "element": "foo",
-  "attributes": {},
+  "element": "string",
```

Comment:
This is not fixed yet. The MSON description of `element` says it should be fixed at value `id` but the example contains value `foo`.